.eslintrc.json

```diff
@@ -17,6 +17,11 @@
     "ecmaVersion": 2021
   },
   "rules": {
```

Comment:
can you please add the `"eol-last": "always"` rule? So eslint will force us to add a new line in every end of files.
ref: https://eslint.org/docs/rules/eol-last